src/lib/support/tests/BUILD.gn

```diff
@@ -60,7 +60,10 @@ chip_test_suite("tests") {
     test_sources += [ "TestCHIPArgParser.cpp" ]
   }
 
-  cflags = [ "-Wconversion" ]
+  cflags = [
+    "-Wconversion",
+    "-Wno-error=uninitialized",
```

Comment:
could we instead make the code work? Maybe use pointers instead of references and maybe use some Init() call?
@msandstedt ?

Reply:
This can be done once the critical failure of the unit tests not building is resolved

Reply:
In that case, maybe we merge this and then keep #21255 open and assign to me.